src/queryResult/queryResultWebViewController.ts

```diff
@@ -233,11 +233,15 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
                 return state;
             }
         });
-        this.registerReducer("addXmlPlan", async (state, payload) => {
-            state.executionPlanState.xmlPlans = [
-                ...state.executionPlanState.xmlPlans,
-                payload.xmlPlan,
-            ];
+        this.registerReducer("openFileThroughLink", async (state, payload) => {
+            // TO DO: add formatting? ADS doesn't do this, but it may be nice...
```

Comment:
make an issue for this, and add the link to the comment so that it can be tracked!  It would be great to get it in for Nov, but perhaps a separate PR to not delay this (more important) one